addons/mail/models/mail_thread.py

```diff
@@ -443,6 +439,24 @@ def _check_can_update_message_content(self, message):
         if not message.message_type == 'comment':
             raise exceptions.UserError(_("Only messages type comment can have their content updated"))
 
+    def _set_default_fields(self, fields):
```

Comment:
```suggestion
    def _set_default_fields(self, fields_list):
```
"fields" is the one from "odoo import fields"
thus, in my opinion is better to do like in default_get (https://github.com/odoo/odoo/blob/15.0/odoo/models.py#L1370)